sdk/src/logs/logger_provider.cc

```diff
@@ -44,19 +46,13 @@ nostd::shared_ptr<logs_api::Logger> LoggerProvider::GetLogger(nostd::string_view
 
```

Comment:
The search logic also needs changes. Probably storing in the `vector` instead of `map`, and comparing for instrumentation_library ( as done in tracer ) along with the logger_name to get the right logger.